ddtrace/tracer/abandonedspans_test.go

```diff
@@ -350,7 +350,7 @@ func TestDebugAbandonedSpansOff(t *testing.T) {
 
 	t.Run("default", func(t *testing.T) {
 		assert := assert.New(t)
-		assert.False(tracer.config.debugAbandonedSpans)
+		assert.False(tracer.config.internalConfig.DebugAbandonedSpans())
```

Comment:
(For here and every other place) is it ever possible for `internalConfig` on the `tracer.config` to be nil? Or for `config` itself to be `nil`?